### DISCUSSION TITLE
Statically compile to include Python

### DISCUSSION THREAD

**Discussion by pax-k** [d99kris/spacy-cpp#25] [Q&A]
Hey!

Thank you for your effort on spacy-cpp! 

I'm trying to wrap spacy-cpp as a React Native Module, but I cannot run it unless Python + SpaCy lib + SpaCy model are bundled together with spacy-cpp.

Are you kind to offer some high level steps about how I could do it? (note: I'm not a C++ dev, but I'll figure stuff out)

Thank you

**Comment by d99kris** [comment]
Hi @pax-k - sorry I don't know exactly how to go about doing that. A similar question has come up before - see #8 - perhaps that discussion can give some pointers. But generally speaking it's unfortunately out of scope for this project.
